packages/x-codemod/src/v8.0.0/charts/preset-safe/expected.spec.tsx

```diff
@@ -27,13 +29,15 @@ function App() {
   <div>
   <PieChart
     slotProps={{
-      legend: { hidden: true }
-    }} />
+      legend: {}
+    }}
+    hideLegend={true} />
```

Comment:
Do you think it's possible to either:

- Do not add a codemod, but modify the `transformLegendToSlots` such that it removes `hide` and add do not create `slotProps.legend` if it was the only property
- Or keep this codemod, but remove `slotPros.legend` and `slotProps` if they are empty

Reply:
For the first solution, you have a helper `removeObjectProperty` that could be inspirational

Reply:
Sure, I'll try that